transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/OpeningHandshakeIT.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2007-2014, Kaazing Corporation. All rights reserved.
+ */
+
+package org.kaazing.gateway.transport.wsn;
+
+import org.apache.log4j.PropertyConfigurator;
+import org.apache.mina.core.future.ConnectFuture;
+import org.apache.mina.core.service.IoHandler;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.DisableOnDebug;
+import org.junit.rules.TestRule;
+import org.junit.rules.Timeout;
+import org.kaazing.gateway.transport.IoHandlerAdapter;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+import org.kaazing.mina.core.session.IoSessionEx;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.rules.RuleChain.outerRule;
+
+
+public class OpeningHandshakeIT {
+
+    private final K3poRule k3po = new K3poRule().setScriptRoot("org/kaazing/specification/ws/opening");
+
+    private final TestRule timeout = new DisableOnDebug(new Timeout(5, SECONDS));
+
+    private final WsnConnectorRule connectorRule = new WsnConnectorRule();
+
+    @Rule
+    public final TestRule chain = outerRule(connectorRule).around(k3po).around(timeout);
+
+
+    @BeforeClass
+    public static void initLogging() throws Exception {
+        PropertyConfigurator.configure("src/test/resources/log4j-trace.properties");
```

Comment:
I'm not up to date on the latest MethodExecuteTrace, but could you also use the MethodExecuteTrace as a rule with the given log4j-trace.properties?  I'm not sure what our best practices are for getting consistent logging in the tests.